inbm/dispatcher-agent/dispatcher/source/source_command.py

```diff
@@ -120,7 +120,7 @@ def _handle_app_source_command(
             logger.info(f"Optional GPG key parameters not present in manifest")
         filename = parsed_head.get_children("applicationSource/remove/repo")["filename"]
         application_source_manager.remove(
-            ApplicationRemoveSourceParameters(file_name=filename, gpg_key_name=keyname)
+            ApplicationRemoveSourceParameters(source_list_file_name=filename, gpg_key_name=keyname)
```

Comment:
The change in parameter names should be reflected in the documentation and any external interfaces that rely on these names. [medium]